homeassistant/components/sensor/strings.json

```diff
@@ -300,12 +300,12 @@
   },
   "issues": {
     "state_class_removed": {
-      "title": "{statistic_id} no longer has a state class",
-      "description": ""
+      "title": "State Class removed",
+      "description": "{statistic_id} no longer has a state class"
     },
     "units_changed": {
-      "title": "The unit of {statistic_id} has changed",
-      "description": ""
+      "title": "Units changed",
+      "description": "The unit of {statistic_id} has changed"
```

Comment:
These issues have special handling in the frontend and thus these descriptions now will be lost?

How did you test this? Can you share a screenshot of how it looks now?

Reply:
Aha, I didn't know that! Hence, no, I haven't tested this.

I found three files in the frontend-repo referencing those variables. But I'm not clever enough to figure out how it works. Can you give me a pointer why it is handled that way? :-) Are the empty description-keys translated elsewhere?

![image](https://github.com/user-attachments/assets/e4abc346-eb5c-4d6e-9d80-64a936b947cf)

I was trying to fix these "Empty" translation keys in Lokalise, because I though it was a bug. But maybe it is better that they are left as is, and that I close this PR? :-)

![image](https://github.com/user-attachments/assets/b3b2ea3c-2892-4c32-b18c-5e2ac3f5ccdf)

Sorry for all the questions, I want to contribute, but I'm very new to all this. :-)